### ISSUE TITLE
Heading node added to accessibleOrder but not as a child is hidden in PDOM

### ISSUE THREAD

**Issue by twant** [phetsims/molarity#63]
@zepumph and I were trying to add the soluteComboBoxHeaderNode to the accessibleOrder of the playArea, but it is not visible in the PDOM unless we also add it as a child. @jessegreenberg is this expect behavior?

**Comment by zepumph** [comment]
Here is a simple case @twant and I were able to reproduce in the scenery playground. It looks like parentless nodes are hidden for some reason.

![image](https://user-images.githubusercontent.com/6856943/53771275-6d355280-3e8e-11e9-9b2f-582c66a70bb0.png)

**Comment by jessegreenberg** [comment]
This looks like a scenery bug, I can't think of a reason that the h3 in https://github.com/phetsims/molarity/issues/63#issuecomment-469477700 would be hidden.

**Comment by jessegreenberg** [comment]
I am starting to wonder if this was actually a design decision. Say the accessibleOrder is specified to separate scene graph and AccessibleInstance tree for a node, then that node is removed from the scene graph. It makes sense that the node is hidden for a11y purposes too. Scanning https://github.com/phetsims/scenery-phet/issues/365 for any more info

**Comment by jessegreenberg** [comment]
From https://github.com/phetsims/scenery-phet/issues/365#issuecomment-383186465, internal question and answer:
> >Do we allow including totally unconnected subtrees in the accessible order?

>No, only connected content for now (simplifies implementation, can change later)

**Comment by jessegreenberg** [comment]
From https://github.com/phetsims/scenery-phet/issues/381#issuecomment-391115532, another comment

> > Note, these nodes wouldn't have to be in the sim's scene graph at all. so the scene graph would look like:

> The playArea/controlPanel nodes definitely need to be in the scene graph (otherwise they wouldn't be tied to an accessible display).

**Comment by jessegreenberg** [comment]
I had previously said that adding a node to an accessibleOrder that is not attached to the scene graph was a supported feature, and I now think that is incorrect. Sorry about that! We should update accessibleOrder documentation to list this explicitly, and perhaps discuss whether this is the ideal behavior.

**Comment by jessegreenberg** [comment]
I added a note about this constraint to `accessibleOrder`.

**Comment by zepumph** [comment]
Oh my goodness thank you for tracking this down! I see now the reasoning for this originally. I can foresee this as a code smell, especially in Molarity, where the screen view is very well organized, and has a list of visual children all added at the end. It would be nice if PDOM Nodes weren't in that list, and only needed to be added through `accessibleOrder` setters on the playAreaNode/controlAreaNode. 

I say this though without a good understanding of how much work it would be to implement the enhancement. No matter the difficulty, it seems like this isn't our highest priority work right now. I'm going to close this issue and we can track it down later when we find updating accessibleOrder to support parentless Nodes is a higher priority.

**Comment by zepumph** [comment]
@jessegreenberg please reopen if you agree.
